AndroidHomeApplication/app/src/main/java/com/example/androidhomeapplication/fragments/movieDetails/FragmentMovieDetails.kt

```diff
@@ -48,6 +45,39 @@ class FragmentMovieDetails : Fragment(R.layout.fragment_movie_details) {
         }
     }
 
+    private fun setupObservers() {
+
+        viewModel.movie.observe(
+            viewLifecycleOwner,
+            {result: DataResult<Movie> ->
```

Comment:
to some private fun
and use link to fun
viewModel.movie.observe(viewLifecycleOwner, ::someAmazingFun)